# Define the goal
float64 initialPhase
float64 strideTime
float64 strideHeight
---
# Define the result
int32 stepsTaken
---
# Define a feedback message
float64 currentPhase
string stage
hexapod_control/Pose currentPose
hexapod_control/Pose targetPose